pkg/rancher-desktop/backend/lima.ts

```diff
@@ -1588,6 +1619,8 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
           await this.kubeBackend.deleteIncompatibleData(kubernetesVersion);
         }
 
+        await this.progressTracker.action('Configuring openresty', 50, this.configureOpenResty(config));
```

Comment:
Maybe call it "registry proxy" instead of "openresty"?

Reply:
Done